sdk/storage/Azure.Storage.Files.Shares/src/ShareServiceClient.cs

```diff
@@ -645,8 +646,100 @@ internal ShareServiceClient(Uri serviceUri, HttpPipelinePolicy authentication, S
 
         #region CreateShare
         /// <summary>
-        /// The <see cref="CreateShare"/> operation creates a new share
-        /// under the specified account. If a share with the same name
+        /// The <see cref="CreateShare(string, ShareCreateOptions, CancellationToken)"/>
+        /// operation creates a new share under the specified account. If a share with the same name
+        /// already exists, the operation fails.
+        ///
+        /// For more information, see
+        /// <see href="https://docs.microsoft.com/rest/api/storageservices/create-share">
+        /// Create Share</see>.
+        /// </summary>
+        /// <param name="shareName">
+        /// The name of the share to create.
+        /// </param>
+        /// <param name="options">
+        /// Optional parameters.
+        /// </param>
+        /// <param name="cancellationToken">
+        /// Optional <see cref="CancellationToken"/> to propagate
+        /// notifications that the operation should be cancelled.
+        /// </param>
+        /// <returns>
+        /// A <see cref="Response{ShareClient}"/> referencing the newly
+        /// created share.
+        /// </returns>
+        /// <remarks>
+        /// A <see cref="RequestFailedException"/> will be thrown if
+        /// a failure occurs.
+        /// </remarks>
+        public virtual Response<ShareClient> CreateShare(
+            string shareName,
+            ShareCreateOptions options,
```

Comment:
if `options` is optional, should it be `ShareCreateOptions options = default` ?

Reply:
I think we need this to prevent ambiguity with the other overloads.